app/validation/validators.py

```diff
@@ -73,9 +75,10 @@ def __init__(self, minimum=None, minimum_exclusive=False,
         self.maximum = maximum
         self.minimum_exclusive = minimum_exclusive
         self.maximum_exclusive = maximum_exclusive
-        if not messages:
-            messages = error_messages
-        self.messages = messages
+        if messages:
```

Comment:
Could be simplified to:

`self.messages = messages or error_messages`

Reply:
nice